Flutter_Weather/lib/widgets/weather_info_day.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_weather/models/index.dart';
+import 'package:flutter_weather/models/weather_extension.dart';
+import 'package:flutter_weather/packages/dafluta/dafluta.dart';
+import 'package:flutter_weather/widgets/icons/weather_type_icon.dart';
+
+class WeatherInfoDay{
```

Comment:
Why this class only have static functions?
This should be a widget class. Constructor will have tow params WeatherInfoDay(this.dayData, this.initDate).